public/api/conf/2.0/application.yaml

```diff
@@ -0,0 +1,261 @@
+openapi: 3.0.3
+info:
+  title: CTC Guarantee Balance
+  contact: {}
+  version: '2.0'
+  description: |
+    # Overview
+    Use the CTC Guarantee Balance API to provide traders with up-to-date
+    information about how much of their guarantee funds they have left to
+    use when organising transit movements.
+servers:
+  - url: https://test-api.service.hmrc.gov.uk/customs/guarantees
+    description: Sandbox
+  - url: https://api.service.hmrc.gov.uk/customs/guarantees
+    description: Production
+    variables: {}
+paths:
+  /customs/guarantees/:grn/balance:
+    post:
+      summary: Get a balance
+      description: Get the balance associated with a guarantee reference number (GRN).
+      operationId: getGuaranteeBalance
+      parameters:
+        - $ref: '#/components/parameters/acceptHeader'
+        - $ref: '#/components/parameters/contentTypeHeader'
+        - in: path
+          name: grn
+          schema:
+            type: string
+            pattern: '^[0-9]{2}[A-Z]{2}[A-Z0-9]{12}[0-9]([A-Z][0-9]{6})?$'
+          required: true
+          description: The guarantee reference number (GRN) of the guarantee.
+      requestBody:
+        description: ''
```

Comment:
I would be inclined to remove this description - there should be no linter errors.

Reply:
Done